src/main/java/io/cdap/delta/bigquery/BigQueryAssessor.java

```diff
@@ -85,7 +85,7 @@ public TableAssessment assess(StandardizedTableDetail tableDetail) {
                     String.format("Table '%s' in database '%s' must have a primary key in order to be replicated",
                                   tableName, dbName),
                     "Please alter the table to use a primary key, or select a different table",
-                    "Not able to replicate this table in BigQuery side"));
+                    "Not able to replicate this table in BigQuery"));
```

Comment:
nit: I think it's also better to say 'to BigQuery' instead of 'in BigQuery'

Reply:
sure. will update before merging.